application-forum-ui/src/main/resources/ForumCode/Actions.xml

```diff
@@ -65,26 +67,34 @@
         $response.setStatus(403)
       #end
     #end
-  #elseif("$!request.action" == "addtopic")
+  #elseif("$!request.action" == 'addtopic')
     #if("$!request.title.trim()" != '')
       #if("$!request.space" != '')
-        #set($newTopicName = $request.page)
```

Comment:
I'm thinking about checking the existence of a Topic and only if exists, add a _number at the end. This way most of the topics will have a clean name added by the user. WDYT?